assignments/assignment_1/index.js

```diff
@@ -1,13 +1,27 @@
 function getNameFromCommandLine() {
     // Write you code here, name should be taken as args in process.argv
+    const input = process.argv;
+    return input[input.length-1];
 }
 
 function getNameFromEnv() {
     // Write your code here
+    process.env.name="Yash"
```

Comment:
no need to declare the environment var "name" here it is already declared in the environment for test cases, just returning is sufficient.

Reply:
yes it's working without declaring also and I'll take care of it from next time, is there any thing do I need to changes except this?